android/guava/src/com/google/common/graph/EndpointPair.java

```diff
@@ -104,6 +104,7 @@ public final N nodeV() {
    * Returns the node that is adjacent to {@code node} along the origin edge.
    *
    * @throws IllegalArgumentException if this {@link EndpointPair} does not contain {@code node}
+   * @since 20.0 (requires that the argument have type {@code N} since 31.0)
```

Comment:
```suggestion
   * @since 20.0 (requires that the argument has type {@code N} since 31.0)
```

Reply:
I am trying to decide if I can justify what I have as technically a valid option. (Compare "requires that the argument be of type N," similar to [in `Queues`](https://github.com/google/guava/blob/1799bc3e48db5ea61ba66191096727d17d1e811c/guava/src/com/google/common/collect/Queues.java#L191) which is maybe also valid??) But I am in over my head grammatically :) I grant that this is at least awkward. Maybe I'll tweak it; unsure.

Reply:
I think the `Queues` alternative is also valid, while I'm not sure I've seen this construct before. But I'm not a native speaker, so... :speak_no_evil:

Reply:
https://github.com/google/guava/pull/5572

Reply:
Way late but just to mention "requires that it have" is valid—it's the [subjunctive mood](https://en.wikipedia.org/wiki/English_subjunctive)